lib/everypolitician/popolo/person.rb

```diff
@@ -42,7 +84,19 @@ def wikidata
       end
 
       def sort_name
-        name
+        document[:name]
```

Comment:
This one will need to also cope with the case of someone actually having a `sort_name` field set.